libs/eo/metering-points/shell/src/lib/eo-metering-points-shell.component.ts

```diff
@@ -35,23 +35,24 @@ const selector = 'ett-dashboard-shell';
         display: block;
       }
       .${selector}__p {
-        @include watt.typography-watt-text-m; // This overrides the styles applied from Angular Material on p tags
+        @include watt.typography-watt-text-m;
       }
     `,
   ],
   template: `
-    <h2>Dashboard</h2>
-    <p class="${selector}__p">
-      More functionality will be released on an ongoing basis. If you want to
-      influence the new functionality, please send an email to
-      <a href="mailto:xkeka@energinet.dk">xkeka@energinet.dk</a>.
-    </p>
+    <p class="${selector}__p">You do not have any metering points.</p>
   `,
+  encapsulation: ViewEncapsulation.None,
```

Comment:
suggestion: We can stop using `ViewEncasulation.None` and instead rely on emulated view encapsulation (the default setting). Same for other components introduced by this pull request.